include/secp256k1.h

```diff
@@ -599,9 +602,10 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_ec_pubkey_negate(
 
 /** Tweak a private key by adding tweak to it.
  * Returns: 0 if the tweak was out of range (chance of around 1 in 2^128 for
- *          uniformly random 32-byte arrays, or if the resulting private key
- *          would be invalid (only when the tweak is the complement of the
- *          private key). 1 otherwise.
+ *          uniformly random 32-byte arrays, or if the given private key is
+ *          invalid according to secp256k1_ec_seckey_verify, or if the resulting
+ *          private key would be invalid (only when the tweak is the complement
```

Comment:
Maybe "negated private key" instead of "complement"?

Is this really the only case? What about overflows?

Reply:
This should also document what the valid range for a tweak is? (I guess every scalar?)

Reply:
Also, missing an `)` after "32-byte arrays". (This was a preexisting problem but we should fix it now.)

Reply:
@apoelstra should be fixed in the latest commit.